assets/css/cart.css

```diff
@@ -0,0 +1,97 @@
+/* Cart Style */
+body {
+  background-image: url("../img/background.jpg");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+}
+
+main {
+  padding: var(--spacing-large);
+  display: flex;
+  justify-content: center;
+}
+
+.total-price {
+  text-align: left;
+  font-size: var(--font-size-large);
+  font-weight: 400;
+  position: absolute;
+  top: 177px;
+  right: 60px;
+  align-items: end;
+  color: var(--color-primary-main);
+}
+
+.price {
+  font-weight: 700;
+  color: var(--color-primary-main);
+}
+
+.cart-list {
+  background-color: var(--color-background);
+  padding: var(--spacing-large);
+  border-radius: 10px;
+  box-shadow: var(--box-shadow-default);
+  width: 100%;
+  max-width: 1804px;
+}
+
+.cart-list h2 {
+  text-align: left;
+  font-size: var(--font-size-xxlarge);
+  font-weight: 700;
+  color: var(--color-primary-main);
+  margin: 15px 5px;
+}
+
+.product-grid {
+  display: grid;
+  grid-template-columns: 0.2fr 2fr 1fr 0.3fr 2.7fr 0.1fr;
+  row-gap: var(--spacing-large);
+  column-gap: var(--spacing-large);
+  align-items: start;
+  padding: var(--spacing-large);
+}
+
+.separator {
+  height: 1px;
+  background-color: var(--color-dark);
+  margin: 5px 0;
+  grid-column: 1 / -1;
+}
+
+.product-header {
+  font-weight: 600;
+  color: var(--color-text-secondary);
+  font-size: var(--font-size-large);
+}
+
+.product-image {
+  width: 121px;
+  height: 116px;
+  background-color: var(--color-dark);
+  border-radius: var(--border-radius-medium);
+}
```

Comment:
Some styles (specifically the table styles) are duplicated in the cart and homepage. You can refactor it, because if you want to change a style for the table, you don't need to change it in two files.

> [!TIP]
> You can create a `table.css` file in import it in cart and homepage style files.